cmd/config/metrics.yml

```
@@ -22,10 +22,10 @@
 
 # Containers & pod metrics
 
-- query: (sum(irate(container_cpu_usage_seconds_total{name!="",container!~"POD|",namespace=~"openshift-(etcd|.*apiserver|ovn-kubernetes|network-node-identity|multus|sdn|ingress|.*controller-manager|.*scheduler)|cilium"}[2m]) * 100) by (container, pod, namespace, node)) > 0
+- query: (sum(irate(container_cpu_usage_seconds_total{name!="",container!~"POD|",namespace=~"openshift-(etcd|.*apiserver|monitoring|ovn-kubernetes|network-node-identity|multus|sdn|ingress|.*controller-manager|.*scheduler)|cilium"}[2m]) * 100) by (container, pod, namespace, node)) > 0
```

As discussed in slack we can apply a filter to the query to discard node-exporter pods

```suggestion
- query: (sum(irate(container_cpu_usage_seconds_total{name!="",name!~="node-exporter-.*", container!~"POD|",namespace=~"openshift-(etcd|.*apiserver|monitoring|ovn-kubernetes|network-node-identity|multus|sdn|ingress|.*controller-manager|.*scheduler)|cilium"}[2m]) * 100) by (container, pod, namespace, node)) > 0
```

---

Ok!, I'll merge this as soon as I see the uwm as well ;)